worker/provisioner/provisioner_task.go

```diff
@@ -1098,11 +1103,6 @@ func (task *provisionerTask) startMachine(
 		return err
 	}
 
-	// TODO (jam): 2017-01-19 Should we be setting this earlier in the cycle?
-	if err := machine.SetInstanceStatus(status.Provisioning, "starting", nil); err != nil {
```

Comment:
Driveby or actually a fix?

Reply:
Drive-by; just moved it up as the first action when adding a machine.